Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseOpMode.java

```diff
@@ -1,57 +1,139 @@
 package org.firstinspires.ftc.team6220_PowerPlay;
 
+import com.qualcomm.hardware.bosch.BNO055IMU;
 import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
 import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.Servo;
 
-abstract public class BaseOpMode extends LinearOpMode {
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder;
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference;
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation;
 
-    //Declared drive motors
-    DcMotor motorFL;
-    DcMotor motorFR;
-    DcMotor motorBL;
-    DcMotor motorBR;
+public abstract class BaseOpMode extends LinearOpMode {
 
-    public void initHardware() {
-        //init drive motors
+    // motors
+    public static DcMotor motorFL;
+    public static DcMotor motorFR;
+    public static DcMotor motorBL;
+    public static DcMotor motorBR;
+
+    //public static DcMotor motorTurntable;
```

Comment:
We were testing on just the chassis and the turntable/slides motors weren't connected. If the motors weren't commented out, there would have been an error with running the opmode because the configuration wouldn't have been correct.